server/index.js

```diff
@@ -0,0 +1,86 @@
+const express = require('express');
+const path = require('path');
+const favicon = require('serve-favicon');
+const logger = require('morgan');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
+
+const db = 'mongodb://localhost/lizardboard'
+
+const routes = require('../routes/index');
+const users = require('../routes/users');
+
+const appRoot = process.env.APP_ROOT
+const app = express();
+
+// view engine setup
+app.set('views', path.join(__dirname, '../views'));
+app.set('view engine', 'pug');
+
+app.set('env', process.env.PORT || '3000')
+if( process.env.NODE_ENV !== 'test' ) app.use( logger( 'dev' ))
+// app.use( cookieSession({
+//   name: 'session',
+//   keys: [[ process.env.SESSION_KEY ]]
+// }))
+// app.use(express.static(buildpath+'/public'))
+app.use(bodyParser.json())
+
+const MongoClient = require('mongodb').MongoClient
+  , assert = require('assert');
+
+// Connection URL
+const url = 'mongodb://127.0.0.1:27017/lizardboard';
+// Use connect method to connect to the Server
+MongoClient.connect(url, function(err, db) {
+  assert.equal(null, err);
+  console.log("Connected correctly to server");
```

Comment:
This is less interesting than if an error occurs - I would remove this and only output something if a failure occurs.  As well, the server probably shouldn't start up unless we manage to connect to the database, so this is probably the wrong place for this code.